packages/lesswrong/lib/types/collectionTypes.ts

```diff
@@ -25,7 +25,7 @@ interface CollectionBase<
   options: CollectionOptions
   addDefaultView: (view: ViewFunction<N>) => void
   addView: (viewName: string, view: ViewFunction<N>) => void
-  defaultView: ViewFunction<N> //FIXME: This is actually nullable (but should just have a default)
+  defaultView?: ViewFunction<N>
```

Comment:
In `packages/lesswrong/lib/generated/viewTypes.ts` above some collections have `ViewTermsBase` as the view type and some have the named collection view terms. This was previously determined by checking if there were any views defined and falling back to `ViewTermsBase` if not.

This ignored the default view though, and for the `Collections` collection we only need a default view, so I have made the default view explicitly nullable (previously it was already nullable but the type was wrong) and added a check for this in the type generation code